internal/driver/cli.go

```diff
@@ -61,6 +64,10 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagContentions := flag.Bool("contentions", false, "Display number of delays at each region")
 	flagMeanDelay := flag.Bool("mean_delay", false, "Display mean delay at each region")
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
+	// HTTPS options
+	flagCertFile := flag.String("cert", "", "TLS client certificate file")
+	flagKeyFile := flag.String("key", "", "TLS private key file")
+	flagCAFile := flag.String("CA", "", "TLS CA certs file")
```

Comment:
It's a bit strange to have command line flag in capital letters. Is there a precedent for this?

Reply:
Agreed.